spec.emu

```diff
@@ -641,7 +682,8 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
             <emu-alg>
               1. If _state_.[[IsLoading]] is *false*, return ~unused~.
               1. If _moduleCompletion_ is a normal completion, then
-                1. Perform InnerModuleLoading(_state_, _moduleCompletion_.[[Value]]<ins>, _phase_</ins>).
+                1. <ins>Let _moduleRequest_ be the ModuleRequest Record in _referrer_.[[RequestedModules]] whose [[Specifier]] is _specifier_.</ins>
```

Comment:
I think this needs to be something more like:

```suggestion
                1. <ins>Let _phase_ be the greater [[Phase]] of all ModuleRequest Records in _referrer_.[[RequestedModules]] whose [[Specifier]] is _specifier_.
```

Reply:
Oh right! I wasn't thinking about duplicate ModuleRequests when writing this. For now I will hard-code "greatest", but once we will have multiple phases (i.e. also instance reflection, or defer imports) all together we will probably need a `ComparePhases(_a_, _b_): ~lt~ | ~gt~ | ~eq~` AO.